### ISSUE TITLE
Allow for a specific number of legend entries

### ISSUE THREAD

**Issue by Dom303** [mwaskom/seaborn#2020]
It would be good if the `legend` entry in `scatterplot` allowed for an integer type that, if the data is numerical, decides how many hues or sizes are used.

I implemented it quite easily, but I don't have push access to the repo.
![image](https://user-images.githubusercontent.com/12683840/78688479-4fbd8e00-78c3-11ea-87be-ab1219941ee9.png)

**Comment by mwaskom** [comment]
I agree that more flexibility here would be nice, but I don't think this is the way to do it. Better, I think, would be passing a Locator object.

**Comment by Dom303** [comment]
I agree, a Locator object would be a nice option
